### PR TITLE
feat(ci): add gcp deploy action

### PR DESCRIPTION
**Pull request by JYisus**
Add GCP deploy workflow migrated from Gitlab CI [https://gitlab.intelygenz.com/konstellation-io/kre-demo](https://gitlab.intelygenz.com/konstellation-io/kre-demo).

### CODE REVIEW

File: .github/workflows/app_deploy-gcp.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  deploy-demo:
+    needs: docker-build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get Version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/}
```
Comment (alpiquero) [inline .github/workflows/app_deploy-gcp.yaml:15]: If the image tags are being generated without the `v` prefix (Ex: `1.0.0` and not `v1.0.0`), should't this line must be like follows?
```
run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/demo-v}
```

File: app/deploy/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: demo-ingress
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    cert-manager.io/issuer: demo-issuer
+#    nginx.ingress.kubernetes.io/whitelist-source-range: "46.27.217.76/32"
```
Comment (alpiquero) [inline app/deploy/ingress.yaml:8]: Remove this comment as it could contain sensitive info.

**Review by alpiquero — CHANGES_REQUESTED** [review]
Some changes requested.

**Comment by aacecandev** [comment]
Is it needed to keep the .vscode folder in the repo?

**Review by aacecandev — CHANGES_REQUESTED** [review]

**Review by aacecandev — APPROVED** [review]

**Pull request merged**